File: api/bilet/bilet.java
package com.oguzhan.api.bilet;

import org.springframework.data.annotation.Id;

import java.sql.Timestamp;

public record bilet(
        @Id
        Integer id,
        Integer sahipid,
        Integer seansid,
        Integer koltukid,
        String alinma_saati,
        String bilettur
) {}

File: veritabaniodevapp/model/bilet.kt
package com.example.veritabaniodevapp.model

import java.io.Serial

data class bilet(
    var id : Int,
    var sahipid: Int,
    var seansid : Int,
    var koltukid : Int,
    var alinma_saati : String,
    var bilettur : String
){
}